File: LeetCode_POTD/June_2026/June_22.py
"""
Easy
TC - O(n), SC - O(1)
"""

class Solution:
    def maxNumberOfBalloons(self, text: str) -> int:
        return min(text.count('l')//2,text.count('o')//2,text.count('a'),text.count('b'),text.count('n'))

File: LeetCode_POTD/June_2026/June_22.java
package Leetcode_POTD.June_2026;

/*
Easy
TC - O(n), SC - O(1)
*/

public class June_22 {
    public int maxNumberOfBalloons(String text) {
        int[]a=new int[26];
        for(int i=0;i<text.length();i++){
            a[text.charAt(i)-'a']++;
        }
        return (int)Math.min(Math.min(a['l'-'a']/2,a['o'-'a']/2),Math.min(a[1],Math.min(a[0],a['n'-'a'])));
    }
}
